src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -250,20 +258,86 @@ public void convert(InputFileWrapper input, OutputFileWrapper output)
                     File relativeOutputDirectory = new File(
                             PathTool.getRelativeFilePath(input.getFile().getAbsolutePath(), f.getParent()));
                     convert(f, input.getEncoding(), input.getFormat(), output, relativeOutputDirectory);
-                    cleanUp(input, f);
                 }
             }
+            try {
+                postProcessAllFiles(output.getFormat());
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+                throw new ConverterException("Error post processing all files: " + e.getMessage(), e);
+            } catch (IOException e) {
+                throw new ConverterException("Error post processing all files: " + e.getMessage(), e);
+            }
```

Comment:
Won't both getMessage() and e be duplicated?

Reply:
Because stacktrace (and cause) is not emitted by default. This pattern is used also e.g. in https://github.com/apache/maven-doxia-converter/pull/87/files/82219f7819769b5e75ef6e0f40e6a23aa875b247#diff-602e7fdec6cff8b7c04882eccbf6704b6be1978a75cf792196d3efde9c9feb3eR249. If we want to change that lets track that individually.